server/paths/refer.ts

```diff
@@ -13,7 +13,7 @@ const personPath = peoplePathBase.path(':prisonNumber')
 const referralsPath = path('/referrals')
 const showReferralPath = referralsPath.path(':referralId')
 const referralPersonPath = showReferralPath.path('person')
-const confirmOasysPath = showReferralPath.path('oasys-confirmed')
+const confirmOasysPath = showReferralPath.path('confirm-oasys')
```

Comment:
I'm a little unsure about this. I suppose `check-answers` does relate to a field as well (`status`) and doesn't use that field as its path, but it has a separate update status endpoint, whereas this is still submitting to the standard referral update path

I suppose it does read better

I wonder if it's worth clarifying in the commit message that this is for the more action-y pages, where you're either doing something or not (confirming/submitting or not), rather than adding data that can take many different forms? Otherwise I wonder if the commit could read as moving towards action-oriented paths for all referral form pages (despite the mention of the reason distinction)

Reply:
Thanks for your thoughts, it's a bit of a grey area for sure, but I've updated the commit message to hopefully clarify this.